templates/serviceaccount.yaml

```diff
@@ -0,0 +1,99 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "cytomine.serviceAccountName" . }}
+  labels:
+    {{- include "cytomine.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: job-creator
+rules:
+- apiGroups: ["batch"]
+  resources: ["jobs", "jobs/status"]
+  verbs: ["get", "watch", "list", "create", "delete"]
+- apiGroups: [""]
+  resources: ["pods", "pods/log", "jobs/log"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: create-jobs
+subjects:
+- kind: ServiceAccount
+  name: {{ include "cytomine.serviceAccountName" . }}
+  apiGroup: ""
+roleRef:
+  kind: Role
+  name: job-creator
+  apiGroup: ""
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ .Release.Name }}-psp
+  annotations:
+   seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+   seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+spec:
+  requiredDropCapabilities:
+    - ALL
+  privileged: true
+  allowPrivilegeEscalation: true
```

Comment:
Is this really needed? does the pods need to access stuff from the host as root?